packages/vuepress-theme-odyssey/.eslintignore

```diff
@@ -0,0 +1 @@
+!.vuepress
```

Comment:
Is the follow-up PR going to remove this and apply the rule fixes?

Reply:
This actually shouldn't be here at all. Will remove.